server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/MessageAppender.java

```diff
@@ -102,14 +109,6 @@ private Keywords keywordsOrDefault(CreationMessage message) {
                 .orElse(Keywords.DEFAULT_VALUE);
     }
 
-    public MessageFactory.MetaDataWithContent appendMessageInMailbox(CreationMessageEntry createdEntry,
```

Comment:
It could have been great to keep that one for fast testing and make it call the new one

Reply:
Couldn't I let the caller call

```
appendMessageInMailboxes(message, ImmutableList.of(mailboxId), session);
```

?

If you wish I'll re-add it, but IMO it have no value

Reply:
As you want, keep it like that